src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/ValidatePackageTask.cs

```diff
@@ -67,7 +62,7 @@ public class ValidatePackageTask : TaskBase
         /// <summary>
         /// Enables strict mode api comparison checks enqueued by the compatible tfm validator.
         /// </summary>
-        public bool EnableStrictModeForCompatibleTfms { get; set; }
+        public bool EnableStrictModeForCompatibleTfms { get; set; } = true;
```

Comment:
That's already the default today, just in the entrypoint target instead of in the task: https://github.com/dotnet/sdk/blob/a7f5021d76002cfc1020715a0c777d9c5f0e7b72/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ApiCompat.ValidatePackage.targets#L48

Reply:
That was introduced with https://github.com/dotnet/sdk/pull/18714#discussion_r663448216

Reply:
Reverted the change here and opened https://github.com/dotnet/sdk/issues/33011 to discuss.